.changeset/many-dancers-flash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/dev-utils': minor
```

Comment:
Not sure if this should be patch instead since it's technically a fix for the previous change that should've been a breaking change or if this should be marked BREAKING since it's also a change in behavior?